src/core/Icon/index.stories.tsx

```diff
@@ -92,10 +93,12 @@ const IconBankWrapper = styled("div")`
 `;
 
 const IconWrapper = styled("div")`
-  ${(props: CommonThemeProps) => {
+  ${(props: CommonThemeProps & { color?: SdsIconColorType }) => {
     const colors = getColors(props);
     const spacings = getSpaces(props);
 
+    const { color } = props;
```

Comment:
should we default this to `primary`?

E.g.,

```ts
const { color = "primary" } = props;

...

background-color: ${colors.[color][400]};
```

Reply:
Aahh, that's much easier 😅